tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -539,7 +541,13 @@ def test_with_dotnetcore(self, runtime, code_uri, mode):
 
         self.verify_docker_container_cleanedup(runtime)
 
-    @parameterized.expand([("dotnetcore2.1", "Dotnetcore2.1"), ("dotnetcore3.1", "Dotnetcore3.1")])
+    @parameterized.expand(
+        [
+            ("dotnetcore2.1", "Dotnetcore2.1"),
+            ("dotnetcore3.1", "Dotnetcore3.1"),
+            ("dotnet5.0", "Dotnet5.0"),
```

Comment:
I believe these integration tests are used for ZIP type building and they are failing in PR checks. I will check if there are image build tests as well.

Reply:
Sure, let me know if any additional change required for tests.